duckduckgo/duckduckgo-privacy-extension
# Remove CTL "surrogate scripts" that now live in tracker-surrogates repo

We have recently moved[1] the Click to Load "surrogate scripts" to the tracker-surrogates[2] repository. Let's complete the work by removing the old copy of those files from this repository.

1 - https://github.com/duckduckgo/tracker-surrogates/commit/54933a370bda2066a9aa05b3a88de7feb287dae0
2 - https://github.com/duckduckgo/tracker-surrogates



**Reviewer:** @jonathanKingston 
**CC:** @ladamski 

## Steps to test this PR:
1. Ensure CTL integration tests still pass.
2. Install extension.
3. Open developer tools and switch to network panel for a new tab.
4. Navigate to https://privacy-test-pages.glitch.me/privacy-protections/click-to-load/
5. Ensure the Facebook SDK request is redirected to the surrogate script successfully.
6. Ensure the test page works OK.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
